### ISSUE TITLE
Correspondence game not timing out.

### ISSUE THREAD

**Issue by CricketForJapan** [gbtami/pychess-variants#1549]
[
](https://www.pychess.org/l9wzT17a)](https://www.pychess.org/l9wzT17a)](https://www.pychess.org/l9wzT17a)

This game is stuck for weeks, and not timing out.

**Comment by CricketForJapan** [comment]
https://www.pychess.org/l9wzT17a

**Comment by gbtami** [comment]
Can you abort it?

**Comment by CricketForJapan** [comment]
Yes I aborted the game.   But the time was stuck. I suppose that they had yet to move so game is void.
